openmetadata-ui/src/main/resources/ui/src/authentication/auth-provider/AuthProvider.interface.ts

```diff
@@ -24,7 +24,7 @@ export type UserProfile = {
   name: string;
   picture: string;
   locale?: string;
-} & Profile;
+} & Pick<Profile, 'preferred_username'>;
```

Comment:
I would recommend to use type from OIDC itself instead maintaining one for newUser

Reply:
Sure, I will check on that.

Reply:
As discussed, we are maintaining the type for OIDC, so we have to use the new type for newUser.